### PR TITLE
feat: improve UI and add some common Chinese characters

### PR DESCRIPTION
**Pull request by yisibl**
## UI changes

- Page Centering
- Jump column Sticky positioning

## Add Chinese characters

- `《出师表》〔两汉〕诸葛亮` Tests for compatibility with the CSS text-spacing-trim property in Chrome(See[ Chrome bug](https://issues.chromium.org/issues/331684447)).  
image
image

- Add more low frequency to Simplified Chinese.
- Usually, web pages contain more characters, so I added a Chinese article to test the real effect.

### CODE REVIEW

File: sample-texts.json [hunk 1]
```diff
@@ -192,6 +197,26 @@
...
+      "龙行龘龘 犄角旮旯<br>",
+      "娉婷袅娜 涕泗滂沱<br>",
+      "呶呶不休 不稂不莠<br>",
+      "咄嗟 蹀躞 耄耋 饕餮<br>",
+      "囹圄 蘡薁 觊觎 龃龉<br>",
+      "狖轭鼯轩 怙恶不悛<br>",
+      "其靁虺虺 腌臢孑孓<br>",
+      "陟罚臧否 针砭时弊<br>",
+      "鳞次栉比 一张一翕<br>",
+       "</span>"
```
Comment (yisibl) [inline sample-texts.json:220]: There are some Chinese characters that can't be displayed and I don't know why.

image

**Comment by garretrieger** [comment]
Apologies, totally missed this. Taking a look now...

File: sample-texts.json (hunk 1 above)
Comment (garretrieger) [reply sample-texts.json:220]: I'm seeing this too, I'll investigate and fix before I merge this.

Comment (garretrieger) [reply sample-texts.json:220]: Ok it's been fixed on main.

**Comment by garretrieger** [comment]
Alright, this looks good to me. Thanks for helping improve the demo!

**Comment by yisibl** [comment]
Thanks!

File: sample-texts.json (hunk 1 above)
Comment (yisibl) [reply sample-texts.json:220]: What's the cause of this?

Comment (garretrieger) [reply sample-texts.json:220]: The IFT font encoding was missing some of the codepoints, I updated to add the missing stuff.

**Pull request merged**